doc/source/ray-air/examples/torch_image_batch_pretrained.py

```diff
@@ -39,4 +39,4 @@ def preprocess(df: pd.DataFrame) -> pd.DataFrame:
 ckpt = TorchCheckpoint.from_model(model=model, preprocessor=preprocessor)
 
 predictor = BatchPredictor.from_checkpoint(ckpt, TorchPredictor)
-predictor.predict(dataset, feature_columns=["image"])
+predictor.predict(dataset, feature_columns=["image"], batch_size=80)
```

Comment:
The default 4096 batch size causes this example to take more than 300 seconds (producing a timeout), added a manual `batch_size` which reduced this down to ~100 seconds.